tensorizer/serialization.py

```diff
@@ -3834,6 +3834,11 @@ def _async_bulk_device_to_host_transfer(
         else:
             transferred = queue.Queue(maxsize=max_read_ahead)
 
+        biggest_tensor_bytes = max(t.nbytes for t in tensors)
```

Comment:
`torch.Tensor.nbytes` is quite new (torch v2.1); this one will be more compatible, and we use it elsewhere in this file
```suggestion
        biggest_tensor_bytes = max(t.element_size() * t.nelement() for t in tensors)
```